packages/opc-feedback/src/opc-feedback.ts

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright (c) 2021 Red Hat One Platform
+ *
+ * MIT
+ *
+ * opc-feedback.ts enables the user to collect the feedback from the user
+ *
+ * @summary opc-feedback.ts will set the properties 
+ * @author Rigin Oommen
+ *
+ * Created at     : 2021-01-18 14:53:24 
+ * Last modified  : 2021-01-25 21:43:54
+ */
+
+import { LitElement, html, property, customElement, internalProperty, query } from 'lit-element';
+import style from './opc-feedback.scss';
+import { repeat } from 'lit-html/directives/repeat.js';
+
+@customElement('opc-feedback')
+export class OpcFeedback extends LitElement {
+  @property({ type: String, attribute: 'url' }) url = '/feedback';
+  @internalProperty() openFeedbackModal = false;
+  @internalProperty() openConfirmationModal = false;
+  @internalProperty() protected summary = '';
+  @internalProperty() experience = '';
+  @internalProperty() category = '';
+  @internalProperty() experienceList = [{
```

Comment:
this is something that should be dynamic. this is a web component and we shouldn't restrict users to use only these values. we can provide some default values but give them an option to override those values as well

Reply:
Implemented default values and provision to override the default tempate values